WordPress/Classes/ViewRelated/Blog/Blog Dashboard/DashboardCard.swift

```diff
@@ -9,6 +9,7 @@ import Foundation
 /// identifier on the backend.
 enum DashboardCard: String, CaseIterable {
     case quickActions
+    case quickStart
```

Comment:
Not necessarily to be done in this PR but you might want to add a property here named `shouldShow`. Then, the `BlogService` will check if this card should really be shown (default is always `true`).

But for this cell, we want to check if there's a quick start enabled in progress or not.

Reply:
Unfortunately enum properties are not settable :(

I think I'll have to inject a Blog into `BlogDashboardService`, so I can call `QuickStartTourGuide.shouldShowChecklist(for: blog)` to check whether quick start is enabled or not.

Reply:
Oh, got it! I was thinking of something like:

```swift
    var shouldShow: Bool {
        switch self {
        case .quickStart:
            return QuickStartTourGuide.shouldShowChecklist(for: blog)
        default:
            return true
        }
    }
```

But I see that we would have to inject the `blog` property, that's an interesting challenged, indeed.

Reply:
I ended up taking the approach I mentioned above. a760e51